static/templates/user_info_popover_content.hbs

```diff
@@ -52,31 +52,36 @@
     </div>
 
     {{#if is_me}}
-        <hr>
-        {{#if can_set_away}}
-        <li>
-            <a href="#" class="set_away_status">
-                <i class="fa fa-minus-circle" aria-hidden="true"></i> {{#tr this}}Set yourself as unavailable{{/tr}}
-            </a>
-        </li>
-        {{/if}}
-        {{#if can_revoke_away}}
-        <li>
-            <a href="#" class="revoke_away_status">
-                <i class="fa fa-minus-circle" aria-hidden="true"></i> {{#tr this}}Set yourself as active{{/tr}}
-            </a>
-        </li>
-        {{/if}}
-        <li>
-            <a href="#" class="update_status_text">
-                <i class="fa fa-comments" aria-hidden="true"></i>
-                {{#if status_text}}
-                    {{#tr this}}Edit status message{{/tr}}
-                {{else}}
-                    {{#tr this}}Set a status message{{/tr}}
-                {{/if}}
-            </a>
-        </li>
+    <hr>
+    <label>Status</label>
+    <select class="status">
+        <option>
```

Comment:
I'm not very good at HTML, so I might be wrong. But I guess you don't need these `<li>` tags anymore and each option, set_status_away and revoke_away_status, should be in their own `<option>` tag.

Reply:
This can be a good resource on how to use the `<select>` and `<option>` tag. https://developer.mozilla.org/en-US/docs/Web/HTML/Element/select